code/game/gamemodes/dynamic/dynamic_rulesets.dm

```diff
@@ -158,7 +166,7 @@
 /// Remember that on roundstart no one knows what their job is at this point.
 /// IMPORTANT: If ready() returns TRUE, that means pre_execute() or execute() should never fail!
 /datum/dynamic_ruleset/proc/ready(forced = 0)
-	if (required_candidates > candidates.len)
+	if (!check_candidates())
 		return FALSE
 	return TRUE
```

Comment:
Why not return `check_candidates()` since it returns true or false?

Reply:
Old vg code that I forgot to fix